charts/vineyard/templates/fuse/daemonset.yaml

```diff
@@ -76,6 +76,8 @@ spec:
             value: {{ .Values.fuse.targetPath }}/rpc-conf
           - name: PRESTOP_MARKER
             value: /tmp/prestop-marker
+          - name: CACHE_SIZE
```

Comment:
How about using 

```yaml
{{- if .Values.fuse.options.cacheSize }}
- name: CACHE_SIZE
  value: "{{ .Values.fuse.options.cacheSize }}"
{{- end }}
```

Reply:
I used a range to avoid specifying each argument one by one.